### DISCUSSION TITLE
how to configure quasar to enable plugin

### DISCUSSION THREAD

**Discussion by jhfoo** [originjs/vite-plugin-federation#360] [Q&A]
Quasar embeds vite config within quasar.config.js, which does not support import statements (`SyntaxError: Cannot use import statement outside a module`)

Does anyone know how to get around this? Quasar uses Vite under the hood.

**Comment by prajapati-mukesh** [comment]
Can any one help on this, I am getting following error:

``` [originjs:federation] Cannot read properties of undefined (reading 'config')
file: virtual:__federation__
C:\Users\amitr\projects\pulse-test-projects\bex-livekit-react\node_modules\@originjs\vite-plugin-federation\dist\index.js:1337
    const serverConfiguration = viteDevServer.config.server;```
